bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/i18n/somfytahoma.properties

```diff
@@ -133,9 +166,25 @@ channel-type.somfytahoma.derogation_heating_mode.description = Derogation heatin
 channel-type.somfytahoma.derogation_heating_mode.command.option.auto = Program
 channel-type.somfytahoma.derogation_heating_mode.command.option.away = Away
 channel-type.somfytahoma.derogation_heating_mode.command.option.comfort = Home
-channel-type.somfytahoma.derogation_heating_mode.command.option.frost\ protection = Frost protection
+channel-type.somfytahoma.derogation_heating_mode.command.option.frost protection = Frost protection
 channel-type.somfytahoma.derogation_heating_mode.command.option.manual = Manual
 channel-type.somfytahoma.derogation_heating_mode.command.option.eco = Night
+channel-type.somfytahoma.dhw.label = DHW Control
+channel-type.somfytahoma.dhw.description = Domestic hot water control
+channel-type.somfytahoma.dhw.state.option.run = Run
+channel-type.somfytahoma.dhw.state.option.stop = Stop
+channel-type.somfytahoma.dhw_mode.label = DHW Mode
+channel-type.somfytahoma.dhw_mode.description = DHW mode control
+channel-type.somfytahoma.dhw_mode.state.option.standard = Standard
+channel-type.somfytahoma.dhw_mode.state.option.high demand = High Demand
```

Comment:
Please add a backslash before the space on the key. 
Please fix all other entries.

Reply:
fixed